app/models/build.js

```diff
@@ -58,6 +59,11 @@ export default Model.extend(DurationCalculations, {
     return jobs.get('length') > 1;
   },
 
+  @computed('tag')
+  isTag(tag) {
+    return (tag && tag.name);
+  },
```

Comment:
Not a big deal but this could use the [`notEmpty`](https://emberjs.com/api/ember/2.15/namespaces/Ember.computed/methods/notEmpty?anchor=notEmpty) computed property.